AuthishModule/ValidationService.cs

```diff
@@ -7,6 +9,7 @@ static class ValidationService
     {
         private static readonly ILog Log = LogManager.GetLogger(typeof(AuthishHandler));
         private static readonly string AuthishPassword = ConfigurationManager.AppSettings["AuthishPassword"];
+        private static readonly string[] AuthishWhitelistedPaths = (ConfigurationManager.AppSettings["AuthishWhitelistedPaths"] ?? "").Split(',');
```

Comment:
It's useful to do a "trim" on the indvidual parts after splitting on ",". You will often have people having configuration with something like "path1, path2", which would give " path2" on the second item, and that would never match.